src/components/Button.js

```diff
@@ -0,0 +1,33 @@
+import styled from "@emotion/styled";
+
+const Button = styled.button`
+  border: none;
+  outline: none;
+  box-shadow: 0px 0px 1px 0px black;
+  padding: ${props => getButtonPadding(props.size)};
+  font-size: ${props => getButtonFontSize(props.size)};
```

Comment:
clarification regarding the earlier comment on `getButtonPadding(size)`.  All this function does is return an existing object and inserting the prop `size`, so you might as well use the object directly. :)
```suggestion
  font-size: ${props => buttonStyles[props.size].fontSize;};
```

you can do the same with `padding` and remove the two function declarations for `fontSize` and `padding`